docs/src/pages/components/geo/index.page.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Geo
+---
+
+import { Example } from '@/components/Example';
+import { Feature } from '@/components/Feature';
+import { Fragment } from '@/components/Fragment';
+import { XStateInspector } from '@/components/XStateInspector';
```

Comment:
Is this needed? We don't have xstate for geo if my understanding is correct.

Reply:
Deleted